README.rst

```diff
@@ -129,16 +249,16 @@ Executing a query
     results = cursor.fetchall()
 
 
-Contributing
-------------
+ Contributing
```

Comment:
is there an added space in front of it, if so please remove it, it's causing formatting issue.